terraform-tests/redis_test.go

```diff
@@ -49,6 +49,7 @@ var _ = Describe("Redis", Label("redis-terraform"), Ordered, func() {
 		"backup_start_hour":                  nil,
 		"backup_end_min":                     nil,
 		"backup_start_min":                   nil,
+		"parameter_group_name":               "fake-param-group-name",
```

Comment:
I don't feel strongly about this, but originally the defaultVars was meant to replicate our brokerpak default values. We would then check we could set other values in specific tests. However we can argue we test defaults property in the cf-tests. If we want faster feedback though, testing our default values work can help us catch missing required values.
We have been slowly deviating from that, just food for thought.